crates/sncast/src/starknet_commands/account/list.rs

```diff
@@ -0,0 +1,61 @@
+use std::collections::HashMap;
+
+use anyhow::anyhow;
+use camino::Utf8PathBuf;
+use clap::Args;
+use indoc::printdoc;
+use itertools::Itertools;
+use sncast::{check_account_file_exists, read_and_parse_json_file, AccountData};
+
+#[derive(Args, Debug)]
+#[command(
+    name = "list",
+    about = "List available accounts",
+    before_help = "Warning! This command exposes cryptographic information, e.g. accounts' private keys"
```

Comment:
How about adding a flag that will determine whether to display the private key or not?